src/registrar/tests/test_admin.py

```diff
@@ -842,12 +837,115 @@ def custom_is_active(self):
         with self.assertRaises(DomainInformation.DoesNotExist):
             domain_information.refresh_from_db()
 
+    def test_has_correct_filters(self):
+        """Tests if DomainApplicationAdmin has the correct filters"""
+        request = self.factory.get("/")
+        request.user = self.superuser
+
+        # Grab the current list of table filters
+        readonly_fields = self.admin.get_list_filter(request)
+        expected_fields = ("status", "organization_type", DomainApplicationAdmin.InvestigatorFilter)
+
+        self.assertEqual(readonly_fields, expected_fields)
+
+    def test_table_sorted_alphabetically(self):
+        """Tests if DomainApplicationAdmin table is sorted alphabetically"""
+        # Creates a list of DomainApplications in scrambled order
+        multiple_unalphabetical_domain_objects("application")
+
+        request = self.factory.get("/")
+        request.user = self.superuser
+
+        # Get the expected list of alphabetically sorted DomainApplications
+        expected_order = DomainApplication.objects.order_by("requested_domain__name")
+
+        # Get the returned queryset
+        queryset = self.admin.get_queryset(request)
+
+        # Check the order
+        self.assertEqual(
+            list(queryset),
+            list(expected_order),
+        )
+
+    def test_displays_investigator_filter(self):
```

Comment:
I don't get what this test is doing that's different than test_has_correct_filters and test_investigator_filter_filters_correctly. I do not mind redundancy in tests so non-blocking, but the 3 tests could use more detailed string descriptions.

Reply:
Will do, thanks. Mostly redundancy in the event the filter is still there but it doesn't sort correctly